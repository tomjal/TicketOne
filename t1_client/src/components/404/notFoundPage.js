import React, { PureComponent } from 'react';

export class NotFoundPage extends PureComponent {
    render() {
        return (
            <div>
                <h2>404 not found</h2>
            </div>
        )
    };
};
