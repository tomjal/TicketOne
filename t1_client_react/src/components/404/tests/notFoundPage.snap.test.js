import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import { NotFoundPage } from './../notFoundPage';

test('NotFoundPage snapshot', () => {
  const wrap = shallow(<NotFoundPage />);

  expect(shallowToJson(wrap)).toMatchSnapshot();
});