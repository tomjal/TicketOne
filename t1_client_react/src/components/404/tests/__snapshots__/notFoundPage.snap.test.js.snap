// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage snapshot 1`] = `
<div>
  <h2>
    404 not found
  </h2>
</div>
`;
